apps/front-client/src/features/explore/hooks/useUpdateCourse.ts

```diff
@@ -29,14 +28,21 @@ export const updateCourse = async (
 };
 
 export const useUpdateCourse = () => {
+  const router = useRouter();
+
   const { data } = useSession();
 
   const userId = data?.user?.sub as string;
   const access_token = data?.access_token;
 
-  const { mutate, isLoading, isError } = useMutation((id: string) =>
-    updateCourse(id, userId, access_token)
+  const { mutate, isLoading, isError } = useMutation(
+    (id: string) => updateCourse(id, userId, access_token),
+    {
+      onSuccess: async () => {
+        await router.push(routes.explore);
```

Comment:
Oula... bah non monsieur tu nous fais quoi là ?
```suggestion
        await queryClient.invalidateQueries({
          queryKey: [COURSE_KEY],
        });
```

Reply:
Oulà !! Pas fait gaffe.